lib/cog/models/bundle_dynamic_config.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Cog.Models.BundleDynamicConfig do
+  use Cog.Model, :no_primary_key
+
+  alias Cog.Models.Bundle
+
+  schema "bundle_dynamic_configs" do
+    field :config, :map
+    field :hash, :string
+
+    belongs_to :bundle, Bundle, [foreign_key: :bundle_id]
+
+    timestamps
+
+  end
+
+  @required_fields ~w(bundle_id config)
+  @optional_fields ~w()
+
+  def changeset(model, params \\ :empty) do
+    model
+    |> cast(params, @required_fields, @optional_fields)
+    |> unique_constraint(:bundle_id, name: :bundle_dynamic_configs_bundle_id_index)
```

Comment:
If the bundle ID is the primary key, would this be better as a field on the bundle table?